apps/slack/slack_win.talon

```diff
@@ -6,15 +6,8 @@ app: slack
 tag(): user.messaging
```

Comment:
above chrome was still chrome.win.talon earlier and this is now slack_win.talon, and then we have win.py. Is this the idea going forward to have app.win.talon or app_win.talon, or should it now be apps/slack/win.talon? i'm pretty indifferent to any of the three, but may as well be consistent

Reply:
The thinking here is:
1. mac.py/win.py => Define all platform-specific actions/lists/whatever
2. chrome.talon => for cross-platform apps, hopefully we take the time to create the actions necessary to have a single .talon file with defined commands. Note there is no chrome.talon at the moment
3. slack_mac.talon => since the .talon files appear in the help system, just naming these "mac.talon" isn't terribly useful. Until we can get around to item 2 for apps like slack, this is probably the least-worst option

Reply:
I guess I'm also open to re-working how the help system works if there's some strong preferences here.

Reply:
thanks for explaining. im fine with that for now really. longer term i guess it would be nice to not specify mac in the help output. i got confused about chrome.win.talon... it was actually a file being deleted.